pkg/snowflake/table.go

```diff
@@ -564,6 +564,7 @@ type table struct {
 	RetentionTime       sql.NullInt32  `db:"retention_time"`
 	AutomaticClustering sql.NullString `db:"automatic_clustering"`
 	ChangeTracking      sql.NullString `db:"change_tracking"`
+	IsExternal          sql.NullString `db:"is_external"`
```

Comment:
I couldn't get it to boolean, sorry. Also, I can't use the WHERE clause because I am using show tables to get tables. I would like to use string ('Y' or 'N') for isexternal to exclude the external table if possible. Alternatively, I could get the table information from the information schema instead of show tables, but that would require a lot of modifications to the table structure.

I'd like to hear your opinion.

Reply:
This should be good for now.